model/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -31,6 +32,11 @@
       <property name="hibernate.format_sql" value="true"/>
       <property name="hibernate.transaction.flush_before_completion" value="true"/>
       <property name="hibernate.id.new_generator_mappings" value="true"/>
+
+      <property name="hibernate.cache.use_query_cache" value="true"/>
+      <property name="hibernate.cache.use_second_level_cache" value="true"/>
+
+      <property name="hibernate.cache.infinispan.statistics" value="true"/>
```

Comment:
does this work out of the box ? 

do we really want to update the 1.1.x branch  ? 

I'd perhap be happy to only work on master - unless we have to :-)

Reply:
@ziccardi any comments?

Reply:
@matzew : yes it works. about the 1.1.x branch, we were doing that on 1.1.x branch when we started this, but I can easily move it to master